bom/build.gradle.kts

```diff
@@ -18,6 +18,7 @@ dependencies {
       "nbt",
       "serializer-configurate3",
       "serializer-configurate4",
+      "text-logger-slf4j",
```

Comment:
I wonder if we perhaps want to make this `text-logger-slf4j1` in line with the configurate serializers and considering the eventual release of slf4j 2?

Reply:
Looking at the alphas for SLF4J 2, they say that the only breaking changes are for logger discovery. For us the change is purely additive -- it'd be adding a layer over the fluent logging system.